<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>708</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>708</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tools Ver:1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="Scanbutton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>端口扫描</string>
   </property>
  </widget>
  <widget class="Line" name="verticalline">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>16</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="horiline">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>310</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OSbutton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>系统识别</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outwindow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>OutputWindow empty now.(Read Only)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputclr">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空输出窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>输出窗口(Output)</string>
   </property>
  </widget>
  <widget class="QLabel" name="input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>输入窗口(Input)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textedit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入参数前请认真阅读使用说明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RestartButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>重新启动程序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExpButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>脚本整合</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InfoButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServiceButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>端口分析</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NullButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>等待更新...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputclr">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空输入窗口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
